sdk/keyvault/keyvault-admin/CHANGELOG.md

```diff
@@ -2,6 +2,18 @@
 
 ## 4.0.0-beta.4 (Unreleased)
 
+- Breaking changes:
+  - Added the "KeyVault" prefix to all of the Key Vault Admin client operations.
+  - Made the AesGcmDecryptParameters authenticationTag required.
+- Collapsed `KeyVaultRoleAssignmentPropertiesWithScope` to `KeyVaultRoleAssignmentProperties`.
+- Renamed `KeyVaultKeyId` to `KeyVaultKeyIdentifier`.
+- Renamed `beginRestore`'s `blobStorageUri` to `folderUri`.
+- Removed `folderName` from `beginRestore`. Now the folder name will be inferred from the `folderUri`.
+- Renamed `beginSelectiveRestore`'s `blobStorageUri` to `folderUri`.
+- Removed `folderName` from `beginSelectiveRestore`. Now the folder name will be inferred from the `folderUri`.
+- Reordered the parameters of `beginSelectiveRestore` to `keyName`, `folderUrl`, `sasToken`, `[options]`.
+- Renamed `KeyVaultBackupResult`'s `backupFolderUri` to `folderUri`.
```

Comment:
Some of these were made on Maor's PR: https://github.com/Azure/azure-sdk-for-js/pull/15246